### DISCUSSION TITLE
Provide a way to Disable Element Resizing by dragging edges with the mouse

### DISCUSSION THREAD

**Discussion by 79ho3ein** [elementor/elementor#21162] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Hi

Currently, by dragging the borders of the elements, it is possible to change their size. This may be very useful, but it may be an annoying feature for some. Because they may accidentally change the size of the elements...

### Describe the solution you'd like

Add a new feature to the User Preferences settings so that we can use it to disable changing the width of elements using drag.

![Screenshot (970)](https://user-images.githubusercontent.com/77163248/209548180-c74e1de8-2f48-4a2d-b263-8ef92bef617c.png)

**Comment by nicholaszein** [comment]
Hello @79ho3ein,

thanks for submitting your feature request! 🙏 

Please understand that while we do welcome new feature requests, we can't promise they will be considered or added to our roadmap for the time being. 

🔖 We will, of course, keep this suggestion in mind and consider adding it if there is significant demand in the future.

_PS: Feature Requests now have a new home, and this is the first issue migrated to Discussions. 🙂 We hope this becomes a better experience for everyone. People can now upvote feature requests! Let your friends know! 😉_ 

Best regards

**Reply by 79ho3ein** [reply]
Hi @nicholaszein, thank you!

Everything is going great for Elementor. Hoping for more success. We support you 💪
